tests/test_parsers.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Flask--Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Parses test module."""
+
+import pytest
+from werkzeug.exceptions import HTTPException
+
+from flask_resources.parsers import (
+    create_request_parser,
+    item_request_parser,
+    search_request_parser,
+)
+
+#
+# Search Request Parser
+#
+
+
+def test_search_request_parser_default_values(app):
+    """Test default search request parser."""
+    with app.test_request_context(
+        "/", method="get", content_type="application/json",
+    ):
+        parsed_args = search_request_parser.parse()
+
+        assert parsed_args.get("size") == 10
+        assert parsed_args.get("page") == 1
+        assert parsed_args.get("q") == ""
+        # pagination values themselves are tested independently
+        assert parsed_args.get("pagination") is not None
```

Comment:
I see `"from"` later but `"from"` is here too, no? I don't see `"page"` having a `"missing"` field.

Reply:
Due to the pagination processing the `from` attribute is not in the returned arguments (`pagination`). However, below `from` is there because it has an invalid value and therefore we are crosschecking that is the one causing the error.

Reply:
About the missing, you are right. `page` was missing `missing` (god bless redundancy 😂 ). `Pagination` was obscuring this because it sets a minimum.

More tests should be added:
- base parser
- custom parser

Reply:
Right I was wondering if there was a minimum set somewhere else. That's good :guitar: